GitCommands/Git/GitCommandHelpers.cs

```diff
@@ -58,45 +58,53 @@ public static class GitCommandHelpers
     {
         private static readonly ISshPathLocator SshPathLocatorInstance = new SshPathLocator();
 
-        public static void SetEnvironmentVariable(bool reload = false)
+        #region Environment variables
+
+        /// <summary>
+        /// The <c>USER</c> environment variable's value for the user/machine.
+        /// </summary>
+        [CanBeNull]
+        private static readonly string UserHomeDir
+            = Environment.GetEnvironmentVariable("HOME", EnvironmentVariableTarget.User)
+           ?? Environment.GetEnvironmentVariable("HOME", EnvironmentVariableTarget.Machine);
```

Comment:
FYI, there is `IEnvironmentAbstraction` to make it easier to inject and test functionality dealing with environment variables

Reply:
Thanks. I'll check it out and get back.